include/driver_entry_interface.h

```diff
@@ -167,6 +176,24 @@ extern "C" {
 /* ========================================================================== */
 
 #ifdef __APPLE__
+#include <stdint.h>
+
+typedef struct bf_ioctl
+{
+    uint32_t command;
+    uint32_t size;
+    void *addr;
+} bf_ioctl_t;
+
+#define IOCTL_ADD_MODULE_LENGTH IOCTL_ADD_MODULE_LENGTH_CMD
```

Comment:
Before we complete 1.1, we should document these, even if the documentation is the same as Linux